src/Nri/Ui/Text/V1.elm

```diff
@@ -138,9 +137,45 @@ captionClassString =
     classString captionClasses
 
 
+classToString : CssClasses -> String
+classToString classes =
+    case classes of
+        Text ->
+            "Text"
+
+        Heading ->
+            "Heading"
+
+        Tagline ->
+            "Tagline"
+
+        SubHeading ->
+            "SubHeading"
+
+        SmallHeading ->
+            "SmallHeading"
+
+        MediumBody ->
+            "MediumBody"
+
+        SmallBody ->
+            "SmallBody"
+
+        SmallBodyGray ->
+            "SmallBodyGray"
+
+        Caption ->
+            "Caption"
+
+
+classToNamespacedString : CssClasses -> String
+classToNamespacedString classes =
+    classToString classes ++ namespace
```

Comment:
💭 Should this be `namespace ++ classToString classes`?

Reply:
Yeah that makes more sense!

The string should be unique either way, but this way would look weird in Inspect Element. 😄 

Changed in 2b11d2acb300e04d4fb6c775197fb6d7fa53ab94

Reply:
Thanks!